haskell/test/resources/transferlist.tz

```diff
@@ -0,0 +1,322 @@
+# SPDX-FileCopyrightText: 2020 Michael J. Klein Tocqueville Group
+# SPDX-License-Identifier: MIT
```

Comment:
`LICENSES/MIT.txt` is not the same file as https://github.com/tqtezos/lorentz-contract-transferlist/blob/master/LICENSE. You should add `LICENSES/LicenseRef-MIT-Klein.txt` and refer to it here.

Reply:
done